packages/manager/src/features/ObjectStorage/BucketLanding/BucketTableRow.tsx

```diff
@@ -50,11 +63,19 @@ export const BucketTableRow = (props: BucketTableRowProps) => {
           <Grid>
             <StyledBucketNameWrapper>
               <Typography component="h3" data-qa-label variant="body1">
-                <StyledBucketLabelLink
-                  to={`/object-storage/buckets/${cluster}/${label}`}
-                >
-                  {label}{' '}
-                </StyledBucketLabelLink>
+                {isObjMultiClusterEnabled ? (
+                  <StyledBucketLabelLink
+                    to={`/object-storage/buckets/${region}/${label}`}
+                  >
+                    {label}{' '}
+                  </StyledBucketLabelLink>
+                ) : (
+                  <StyledBucketLabelLink
+                    to={`/object-storage/buckets/${cluster}/${label}`}
+                  >
+                    {label}{' '}
+                  </StyledBucketLabelLink>
+                )}
```

Comment:
Can we move the conditional enabled check to the `to` prop rather than conditionally rendering the entire link component?

```suggestion
                  <StyledBucketLabelLink
                    to={`/object-storage/buckets/${isObjMultiClusterEnabled ? region : cluster }/${label}`}
                  >
                    {label}{' '}
                  </StyledBucketLabelLink>
```